docker/test/integration/steps/steps.py

```diff
@@ -556,9 +556,9 @@ def step_impl(context, log_message, duration):
     context.test.check_minifi_log_contents(log_message, timeparse(duration))
 
 # MQTT
```

Comment:
This line fails flake8 test locally because 2 blank lines are expected:
`steps.py:559:1: E302 expected 2 blank lines, found 1`

Somehow it does not fail in the CI which should be checked.

Reply:
I checked before submitting and nothing failed for me either. Strange. My local version is:
```
adam@amarkovics-Linux:~/work/minifi-cpp/build$ flake8 --version
3.7.9 (mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.1) CPython 3.8.10 on Linux
```

Reply:
Must be the version difference (in CI as well) because my local version is 3.8.4